libs/sdk-ui-kit/src/Header/generateHeaderAccountMenuItems.ts

```diff
@@ -18,15 +18,17 @@ export function generateHeaderAccountMenuItems(
     uiSettings: IUiSettings, // bootstrapResource.settings
     workspaceId?: string, // parsed from bootstrapResource.current.project.links.self
     showOnlyLogoutItem?: boolean,
+    featureFlags?: ISettings,
 ): IHeaderMenuItem[] {
     const { canInitData } = workspacePermissions;
     const { displayAccountPage } = uiSettings;
     const accountMenuItems: IHeaderMenuItem[] = [];
+    const workspaceRef = featureFlags?.enableRenamingProjectToWorkspace === true ? "workspace" : "project";
```

Comment:
`=== true` is not needed, is it?

Reply:
I have tested with test cases and it's redundant thanks I have changed it